WooCommerce/UITestsFoundation/Screens/Products/ProductsScreen.swift

```diff
@@ -59,18 +48,24 @@ public final class ProductsScreen: ScreenObject {
         return try SingleProductScreen()
     }
 
-    public func verifyProductListOnProductsScreen(count: Int, name: String, status: String) throws -> Self {
-        XCTAssertTrue(try app.getTextVisibilityCount(text: name) == 1, "Product name does not exist!")
-        XCTAssertTrue(app.tables.cells.count == count, "Expecting \(count) products, got \(app.tables.cells.count) instead!")
+    func verifyStockStatusForProduct(name: String, status: String) throws -> Bool {
+        let namePredicate = NSPredicate(format: "identifier == %@", name)
+        let statusPredicate = NSPredicate(format: "label ==[c] %@", status)
```

Comment:
I used to work with a case sensitive language called Gherkin and have spent many hours trying to debug rogue caps issues. From that experience, I found it is very easy to make mistakes when typing in the case. 

Having expected texts to be all lowercase (when we're looking for a specific value not the capitalization) I think would make it easier for maintenance in the long run as there's a possibility that not all texts are capitalized the same way across screens. 

Again, this is something I inherited from past experience. We should definitely discuss this as well so that we can have a more standardized way of writing tests across the apps.